dashboard/app/models/lti.rb

```diff
@@ -0,0 +1,5 @@
+module Lti
+  def self.table_name_prefix
```

Comment:
I may be missing something, could you point out where this method is used?

Reply:
This is a part of the Rails model modularization functionality: [Using table_name_prefix and namespace to organize models](https://icelandcheng.medium.com/ruby-on-rails-using-table-name-prefix-and-namespace-to-organize-models-a08199deaa69)